### DISCUSSION TITLE
pauseClock option in API

### DISCUSSION THREAD

**Discussion by piper93** [vigoren/foundryvtt-simple-calendar#333] [Ideas]
Hi! This module is great but I was wondering if there could be a bit more time control available through the API? The startClock() and stopClock() methods are great, but it would be handy to be able to pause the clock too. Not sure if I've just missed something though, maybe that's already an option!

**Comment by vigoren** [comment]
Hey! You are right there is no way to pause the clock through the API. I overlooked that somehow haha.

I will add a function to pause the clock for the next release!

**Comment by vigoren** [comment]
A new function has been added to the API in version 2.0.20
`SimpleCalendar.api.pauseClock()` which will pause the clock if it is running.

**Reply by piper93** [reply]
That's great, thank you so much!
